sql/driver/driver_test.go

```diff
@@ -273,91 +272,3 @@ func TestInsecure(t *testing.T) {
 		t.Fatal(err)
 	}
 }
-
-// Start a transaction to update a row. After updating the row and
-// before committing the transaction, start another transaction in
-// another thread to update the same row. The other transaction succeeds,
-// resulting in the first transaction failing.
-func TestOverlappingTransactions(t *testing.T) {
```

Comment:
I'm fine with seeing this test removed, but as @tschottdorf points out, we do need tests that we're handling various transaction cases correctly.